src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java

```diff
@@ -38,9 +42,13 @@
 
   @Retryable
   public void sendMessage(Object dto) throws GatewayException {
+    Message gatewayMessage;
```

Comment:
Java code style is declare where is created, so move this to line 48 so it reads 
Message gatewayMessage = convertJSONToMessage(JSONJobRequest);

Reply:
Changed code.